autorelease/tag.py

```diff
@@ -48,16 +52,24 @@ def process_issue(
     # is necessary because github gives us back an issue object, but it
     # doesn't contain all of the PR info.
     pull = gh.get_url(issue["pull_request"]["url"])
+    repo_full_name = pull["base"]["repo"]["full_name"]
```

Comment:
I think repo full name will be `googleapis/google-cloudevents-dotnet`.

Example API response: https://api.github.com/repos/googleapis/releasetool/pulls/579 -> `googleapis/releasetool`

Reply:
Yep, better.